yarn-project/stdlib/src/tx/offchain_effect.ts

```diff
@@ -0,0 +1,19 @@
+import { Fr } from '@aztec/foundation/fields';
+
+import type { AztecAddress } from '../aztec-address/index.js';
+
+// poseidon2hash("aztecnr_offchain_message")
+export const OFFCHAIN_MESSAGE_IDENTIFIER: Fr = new Fr(
```

Comment:
Would add here a comment that it was copied over from Noir and that it has to be in sync.